java/create-geometries/src/main/java/com/esri/arcgisruntime/sample/creategeometries/MainActivity.java

```diff
@@ -43,6 +43,40 @@
  */
 public class MainActivity extends AppCompatActivity {
 
+  private MapView mMapView;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
```

Comment:
Why was the order of methods changed here? This order was deliberate, to show the creation of geometries as the first code in the sample (code guidelines indicate constructors should be before class methods, but this isn't a constructor).

Reply:
oops, ok. Reverted!